README.md

```diff
@@ -587,6 +587,64 @@ list slicing notation:
 new_table = old_table[0:5]
 ```
 
+## Themes
+
+### Getting Themes
+
+If you want your tables to be even fancier (if that's possible) then you can use color
+themes. All you have to do is import it.
+
+```python
+from prettytable import PrettyTable, THEME
+```
+
+> Note, if you are using Windows, type `pip install prettytable[windows]`, so the color
+> codes get reformatted to your OS. If color does not show for other operating systems,
+> you can still do this and that may fix your problem.
+
+### Using Themes
+
+After you are certain that you've imported the theme, you can quickly apply it. Now,
+printing the theme will include color.
+
+```python
+x = PrettyTable()
+x.set_theme(THEME.OCEAN)
+print(x)
+```
+
+You can also set theme on import.
+
+```python
+x = PrettyTable(theme = THEME.OCEAN)
+print(x)
+```
+
+### Custom Themes
+
+If you don't want any of the themes supplied, you can make your own. Create a custom
+dictionary with the following keys:
```

Comment:
Maybe mention what the keys are with a link to a reference guide?

Reply:
What kind of reference? For color codes, custom themes, or what?

Reply:
Just as a follow up, this is the last thing that you wanted me to do before I format and finish. Can you please explain what you mean by this?
Thanks!